p2p/libp2p/netMessenger.go

```diff
@@ -924,6 +934,92 @@ func (netMes *networkMessenger) Broadcast(topic string, buff []byte) {
 	netMes.BroadcastOnChannel(topic, topic, buff)
 }
 
+// BroadcastOnChannelBlockingWithSk tries to send a byte buffer onto a topic using provided channel
+// It is a blocking method. It needs to be launched on a go routine
+func (netMes *networkMessenger) BroadcastOnChannelBlockingWithSk(
+	channel string,
+	topic string,
+	buff []byte,
+	pid core.PeerID,
+	skBytes []byte,
+) error {
+
+	id := peer.ID(pid)
+	sk, err := crypto.UnmarshalPrivateKey(skBytes)
+	if err != nil {
+		return err
+	}
+
+	err = netMes.checkSendableData(buff)
```

Comment:
might think on better namings? because we have this function:
`func (netMes *networkMessenger) publish(topic *pubsub.Topic, data *p2p.SendableData, buffToSend []byte) error {`
which has `sendableData` and `buff`. 
in this line, you assume that a sendableData is buff
later on, on sendableData, you also set a buff

it is confusing. can we rename and use more specific namings instead of buff?

Reply:
renamed `buff` to `packedSendableDataBuff` as it explains better what does the slice contain